### PR TITLE
Integration for configuring workspace bindings

### PR DESCRIPTION
**Pull request by willwwt**
Add get_workspace_bindings and update_workspace_bindings CLI for workspace bindings configuration.

Manual test:
```
(venv2.7) Q7Q6DF6LLJ:databricks-cli Weitao.Wen$ databricks --profile e2-weitao unity-catalog catalogs
Usage: databricks unity-catalog catalogs [OPTIONS] COMMAND [ARGS]...

Options:
  -h, --help  Show this message and exit.

Commands:
  create           Create a new catalog.
  delete           Delete a catalog.
  get              Get a catalog.
  get-bindings     Get workspace bindings of a catalog.
  list             List catalogs.
  update           Update a catalog.
  update-bindings  Update workspace bindings of a catalog.

(venv2.7) Q7Q6DF6LLJ:databricks-cli Weitao.Wen$ databricks unity-catalog catalogs update-bindings --name weitao-catalog --json-file update-bindings.json --profile e2-weitao
{
  "workspaces": [
    6051921418418893
  ]
}
(venv2.7) Q7Q6DF6LLJ:databricks-cli Weitao.Wen$ databricks --profile e2-weitao unity-catalog catalogs get-bindings --name weitao-catalog
{
  "workspaces": [
    6051921418418893
  ]
}
(venv2.7) Q7Q6DF6LLJ:databricks-cli Weitao.Wen$ cat update-bindings.json
{
    "assign_workspaces": [6051921418418893]
}
(venv2.7) Q7Q6DF6LLJ:databricks-cli Weitao.Wen$ vi update-bindings.json
(venv2.7) Q7Q6DF6LLJ:databricks-cli Weitao.Wen$ databricks unity-catalog catalogs update-bindings --name weitao-catalog --json-file update-bindings.json --profile e2-weitao
{}
(venv2.7) Q7Q6DF6LLJ:databricks-cli Weitao.Wen$ cat update-bindings.json
{
    "unassign_workspaces": [6051921418418893]
}
(venv2.7) Q7Q6DF6LLJ:databricks-cli Weitao.Wen$ databricks --profile e2-weitao unity-catalog catalogs get-bindings --name weitao-catalog
{}
```

### CODE REVIEW

**Review by adamcain-db — COMMENTED** [review]
Additions LGTM.  @willwwt said he'd add unit tests.

File: databricks_cli/unity_catalog/catalog_cli.py [hunk 1]
```diff
@@ -137,6 +137,44 @@ def delete_catalog_cli(api_client, name, purge):
     UnityCatalogApi(api_client).delete_catalog(name)
 
 
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get workspace bindings of a catalog.')
+@click.option('--name', required=True,
+              help='Name of the catalog to get.')
```
Comment (yeshengm) [inline databricks_cli/unity_catalog/catalog_cli.py:143]:
```suggestion
              help='Name of the catalog to get bindings for.')
```

File: databricks_cli/unity_catalog/catalog_cli.py [hunk 2]
```diff
@@ -137,6 +137,44 @@ def delete_catalog_cli(api_client, name, purge):
...
+    Get workspace bindings of a catalog.
+    """
+    catalog_json = UnityCatalogApi(api_client).get_catalog_bindings(name)
+    click.echo(mc_pretty_format(catalog_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Update workspace bindings of a catalog.')
+@click.option('--name', required=True,
+              help='Name of the catalog to update.')
```
Comment (yeshengm) [inline databricks_cli/unity_catalog/catalog_cli.py:159]:
```suggestion
              help='Name of the catalog to update bindings for.')
```

**Comment by willwwt** [comment]
> Additions LGTM. @willwwt said he'd add unit tests.

Added unit tests in tests/unity_catalog/test_catalog_cli.py

**Review by adamcain-db — APPROVED** [review]
LGTM

**Review by yeshengm — APPROVED** [review]
LGTM

**Review by pietern — APPROVED** [review]

**Pull request merged**